This is workaround for bnc#856931.
The "None" color is not used and causes the title to be invisible due to bug in Xserver/libgdk.
diff -ur icewm-1.3.8.orig/lib/themes/icedesert/titleAB.xpm icewm-1.3.8/lib/themes/icedesert/titleAB.xpm
--- icewm-1.3.8.orig/lib/themes/icedesert/titleAB.xpm	2014-03-03 14:04:45.087165548 +0200
+++ icewm-1.3.8/lib/themes/icedesert/titleAB.xpm	2014-03-03 14:05:27.926953675 +0200
@@ -1,7 +1,6 @@
 /* XPM */
 static char * titleAB_xpm[] = {
-"1 20 20 1",
-" 	c None",
+"1 20 19 1",
 ".	c #FFFFFF",
 "+	c #FDFBFA",
 "@	c #FBF8F5",
diff -ur icewm-1.3.8.orig/lib/themes/icedesert/titleAT.xpm icewm-1.3.8/lib/themes/icedesert/titleAT.xpm
--- icewm-1.3.8.orig/lib/themes/icedesert/titleAT.xpm	2014-03-03 14:04:45.086165483 +0200
+++ icewm-1.3.8/lib/themes/icedesert/titleAT.xpm	2014-03-03 14:05:36.371503268 +0200
@@ -1,7 +1,6 @@
 /* XPM */
 static char * titleAT_xpm[] = {
-"1 20 20 1",
-" 	c None",
+"1 20 19 1",
 ".	c #FFFFFF",
 "+	c #FDFBFA",
 "@	c #FBF8F5",
diff -ur icewm-1.3.8.orig/lib/themes/icedesert/titleIB.xpm icewm-1.3.8/lib/themes/icedesert/titleIB.xpm
--- icewm-1.3.8.orig/lib/themes/icedesert/titleIB.xpm	2014-03-03 14:04:45.086165483 +0200
+++ icewm-1.3.8/lib/themes/icedesert/titleIB.xpm	2014-03-03 14:05:44.591038217 +0200
@@ -1,7 +1,6 @@
 /* XPM */
 static char * titleIB_xpm[] = {
-"1 20 20 1",
-" 	c None",
+"1 20 19 1",
 ".	c #FFFFFF",
 "+	c #FBFBFB",
 "@	c #F8F8F8",
diff -ur icewm-1.3.8.orig/lib/themes/icedesert/titleIT.xpm icewm-1.3.8/lib/themes/icedesert/titleIT.xpm
--- icewm-1.3.8.orig/lib/themes/icedesert/titleIT.xpm	2014-03-03 14:04:45.086165483 +0200
+++ icewm-1.3.8/lib/themes/icedesert/titleIT.xpm	2014-03-03 14:05:54.318671317 +0200
@@ -1,7 +1,6 @@
 /* XPM */
 static char * titleIT_xpm[] = {
-"1 20 20 1",
-" 	c None",
+"1 20 19 1",
 ".	c #FFFFFF",
 "+	c #FBFBFB",
 "@	c #F8F8F8",
diff -ur icewm-1.3.8.orig/lib/themes/Infadel2/titleAB.xpm icewm-1.3.8/lib/themes/Infadel2/titleAB.xpm
--- icewm-1.3.8.orig/lib/themes/Infadel2/titleAB.xpm	2014-03-03 14:04:45.089165678 +0200
+++ icewm-1.3.8/lib/themes/Infadel2/titleAB.xpm	2014-03-03 14:06:37.815502206 +0200
@@ -1,7 +1,6 @@
 /* XPM */
 static char * titleAS_xpm[] = {
-"1 17 12 1",
-" 	c None",
+"1 17 11 1",
 ".	c #868687",
 "+	c #9A9A9B",
 "@	c #AEAEAF",
diff -ur icewm-1.3.8.orig/lib/themes/Infadel2/titleAS.xpm icewm-1.3.8/lib/themes/Infadel2/titleAS.xpm
--- icewm-1.3.8.orig/lib/themes/Infadel2/titleAS.xpm	2014-03-03 14:04:45.089165678 +0200
+++ icewm-1.3.8/lib/themes/Infadel2/titleAS.xpm	2014-03-03 14:06:43.555875804 +0200
@@ -1,7 +1,6 @@
 /* XPM */
 static char * titleAS_xpm[] = {
-"1 17 12 1",
-" 	c None",
+"1 17 11 1",
 ".	c #868687",
 "+	c #9A9A9B",
 "@	c #AEAEAF",
diff -ur icewm-1.3.8.orig/lib/themes/Infadel2/titleAT.xpm icewm-1.3.8/lib/themes/Infadel2/titleAT.xpm
--- icewm-1.3.8.orig/lib/themes/Infadel2/titleAT.xpm	2014-03-03 14:04:45.088165613 +0200
+++ icewm-1.3.8/lib/themes/Infadel2/titleAT.xpm	2014-03-03 14:06:50.628336098 +0200
@@ -1,7 +1,6 @@
 /* XPM */
 static char * titleAT_xpm[] = {
-"1 17 18 1",
-" 	c None",
+"1 17 17 1",
 ".	c #868687",
 "+	c #484848",
 "@	c #344066",
diff -ur icewm-1.3.8.orig/lib/themes/Infadel2/titleIB.xpm icewm-1.3.8/lib/themes/Infadel2/titleIB.xpm
--- icewm-1.3.8.orig/lib/themes/Infadel2/titleIB.xpm	2014-03-03 14:04:45.089165678 +0200
+++ icewm-1.3.8/lib/themes/Infadel2/titleIB.xpm	2014-03-03 14:06:56.526719980 +0200
@@ -1,7 +1,6 @@
 /* XPM */
 static char * titleAS_xpm[] = {
-"1 17 12 1",
-" 	c None",
+"1 17 11 1",
 ".	c #868687",
 "+	c #9A9A9B",
 "@	c #AEAEAF",
diff -ur icewm-1.3.8.orig/lib/themes/Infadel2/titleIS.xpm icewm-1.3.8/lib/themes/Infadel2/titleIS.xpm
--- icewm-1.3.8.orig/lib/themes/Infadel2/titleIS.xpm	2014-03-03 14:04:45.089165678 +0200
+++ icewm-1.3.8/lib/themes/Infadel2/titleIS.xpm	2014-03-03 14:07:03.445170251 +0200
@@ -1,7 +1,6 @@
 /* XPM */
 static char * titleAS_xpm[] = {
-"1 17 12 1",
-" 	c None",
+"1 17 11 1",
 ".	c #868687",
 "+	c #9A9A9B",
 "@	c #AEAEAF",
diff -ur icewm-1.3.8.orig/lib/themes/Infadel2/titleIT.xpm icewm-1.3.8/lib/themes/Infadel2/titleIT.xpm
--- icewm-1.3.8.orig/lib/themes/Infadel2/titleIT.xpm	2014-03-03 14:04:45.089165678 +0200
+++ icewm-1.3.8/lib/themes/Infadel2/titleIT.xpm	2014-03-03 14:07:09.963594488 +0200
@@ -1,7 +1,6 @@
 /* XPM */
 static char * titleIT_xpm[] = {
-"1 17 18 1",
-" 	c None",
+"1 17 17 1",
 ".	c #868686",
 "+	c #484848",
 "@	c #4D4D4D",
